src/twisted/words/service.py

```diff
@@ -980,7 +980,7 @@ class PBGroupReference(pb.RemoteReference):
     def unjellyFor(self, unjellier, unjellyList):
         clsName, name, ref = unjellyList
         self.name = name
-        if bytes != str and isinstance(self.name, bytes):
+        if isinstance(self.name, bytes):
```

Comment:
Is this `if` statement needed?

The current tests alwasy pass name as bytes

Reply:
I removed the line. I did not see clearly where this is tested explicitly though (didn't see `PBGroupReference` in the tests).